### PR TITLE
R and CRAN packages update

### PR DESCRIPTION
**Pull request by wagnerflo**

### CODE REVIEW

File: srcpkgs/R-cran-ggplot2/template [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
 		R-cran-reshape2 R-cran-scales>=0.4.1 R-cran-tibble
-		R-cran-lazyeval"
+		R-cran-lazyeval R-cran-withr>=2.0.0"
 makedepends+="	R-cran-digest R-cran-gtable R-cran-plyr
 		R-cran-reshape2 R-cran-scales R-cran-tibble
-		R-cran-lazyeval"
+		R-cran-lazyeval R-cran-withr"
 short_desc="Create Elegant Data Visualisations Using the Grammar of Graphics"
 maintainer="Florian Wagner <florian@wagner-flo.net>"
 license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/R-cran-ggplot2/template]: please use SPDX identifier

File: srcpkgs/R-cran-stringi/template [hunk 2]
```diff
@@ -1,14 +1,14 @@
 # Template file for 'R-cran-stringi'
 pkgname=R-cran-stringi
-version=1.2.2
+version=1.2.3
 revision=1
 build_style=R-cran
 makedepends+=" pkg-config icu-devel"
 short_desc="Character String Processing Facilities"
 maintainer="Florian Wagner <florian@wagner-flo.net>"
 license="BSD"
```
Comment (maxice8) [inline srcpkgs/R-cran-stringi/template]: please use SPDX identifier

File: srcpkgs/R-cran-withr/template [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Template file for 'R-cran-withr'
+pkgname=R-cran-withr
+version=2.1.2
+revision=1
+build_style=R-cran
+short_desc="Run Code 'With' Temporarily Modified Global State"
+maintainer="Florian Wagner <florian@wagner-flo.net>"
+license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/R-cran-withr/template]: please use SPDX identifier

**Comment by wagnerflo** [comment]
Here you go!

File: srcpkgs/R/template [hunk 4]
```diff
@@ -19,7 +19,7 @@ license="GPL-2.0-or-later"
 homepage="http://www.r-project.org/"
```
Comment (maxice8) [inline srcpkgs/R/template]: this is https://

File: srcpkgs/R/template [hunk 5]
```diff
@@ -19,7 +19,7 @@ license="GPL-2.0-or-later"
...
 #changelog="https://cran.r-project.org/doc/manuals/r-release/NEWS.html"
```
Comment (maxice8) [inline srcpkgs/R/template]: this is now an accepted field

File: srcpkgs/R/template [hunk 6]
```diff
@@ -19,7 +19,7 @@ license="GPL-2.0-or-later"
...
 distfiles="http://cran.r-project.org/src/base/R-3/${pkgname}-${version}.tar.gz"
```
Comment (maxice8) [inline srcpkgs/R/template]: this should be https:// too

**Comment by wagnerflo** [comment]
This should do now...

**Comment by maxice8** [comment]
It does

**Pull request closed without merging**